static/site.css

```diff
@@ -5,21 +5,71 @@ body {
     font-size: 12pt;
 }
 
```

Comment:
```suggestion

img.logo {
   vertical-align: middle
}

```

Reply:
I added the class, but needed to specify it different for large vs small screens.